src/utils/dialect.ts

```diff
@@ -0,0 +1,69 @@
+import isPlainObject from 'lodash/isPlainObject';
+// eslint-disable-next-line import/order -- caused by temporarily mixing require with import
+import { v1 as uuidv1, v4 as uuidv4 } from 'uuid';
+
+const DataTypes = require('../data-types');
+
+const dialects = new Set([
```

Comment:
Not sure. This variable name is terrible because based on usage, it looks like it's used to mean "dialects that support milliseconds in their datatypes"

Reply:
Yes, let's update this here. Or at least add a comment to this

Reply:
Done! I've also added a comment on how to rework this function when we work on splitting dialects from core